.github/workflows/tools.yml

```diff
@@ -77,6 +77,16 @@ jobs:
                 echo "NEW_VERSION=$NEW_VERSION" >> $GITHUB_ENV
                 ./tools/update-undici.sh
               fi
+          - id: postject
+            subsystem: deps
+            label: dependencies
+            run: |
+              NEW_VERSION=$(npm view postject dist-tags.latest)
+              CURRENT_VERSION=$(node -p "require('./test/fixtures/postject-copy/node_modules/postject/package.json').version")
+              if [ "$NEW_VERSION" != "$CURRENT_VERSION" ]; then
+                echo "NEW_VERSION=$NEW_VERSION" >> $GITHUB_ENV
+                ./tools/update-postject.sh
+              fi
```

Comment:
WDYT about having this logic in the `tools/update-postject.sh` script too?

I understand the separation of concerns here, but you probably don't want to update the version if it's unnecessary.

Reply:
I actually copy-pasted this part from the undici updater script which is right above this one. I didn't add it to `tools/update-postject.sh` because the `tools/update-undici.sh` script doesn't have it. What do you think about doing this for all the updater scripts in one shot in a separate PR?

Reply:
> What do you think about doing this for all the updater scripts in one shot in a separate PR?
+1 from me on that suggestion.

Reply:
> What do you think about doing this for all the updater scripts in one shot in a separate PR?

100% agree!